js/bccm-example-config.js

```diff
@@ -17,71 +17,71 @@ export default {
                     {
                         label: `test1`,
                         processId: `process_f50625b5-0c30-4364-b26e-4ec0f01db436`,
-                        button_colour: colour.yellow,
+                        buttonColor: color.yellow,
```

Comment:
Welp, this is not a good idea, since the config files are now already out of your control. You should not change the syntax of the config files in a non-backwards compatible way.